examples/stress_tests/particles.rs

```diff
@@ -170,7 +170,6 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         app.rollback_component_with_reflect::<Sprite>()
             .rollback_component_with_reflect::<Transform>()
             .rollback_component_with_reflect::<GlobalTransform>()
-            .rollback_component_with_reflect::<Handle<Image>>()
```

Comment:
Handles as component were removed in 0.15, but where did this particular handle go?

Reply:
In Sprite